src/components/base-address/interfaces/address-interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface AddressIF {
+  street: string
+  streetAdditional?: string
+  city: string
+  region: string
+  postalCode: string
+  country: string
+  deliveryInstructions?: string
+}
```

Comment:
Is this an internal-only interface? If not then it should be in the interfaces folder so that UIs can import it. (There may be an address interface there already. Can it be reused?)

Reply:
Fixed, thanks!